armi/tests/test_notebooks.py

```diff
@@ -36,6 +36,9 @@ def test_runParamSweep(self):
 
     def test_runDataModel(self):
         runNotebook(os.path.join(TUTORIALS, "data_model.ipynb"))
+        # Do some cleanup because some code run in the notebook doesn't honor the
```

Comment:
Can you wrap this in a temporary directory changer before calling the jupyter notebook? Maybe within the `runNotebook` function?

Reply:
I tried that first. I think some places in the code use absolute file paths and I don't want to change the code unless there's a good reason to